inspector/web/components/app-history.css

```diff
@@ -1,6 +1,6 @@
 app-history {
   border-left: var(--panel-border);
-  padding: var(--space-lg);
+  padding: var(--space-lg) 0;
```

Comment:
Moved the horizontal padding to the child elements so that the scroll bar isn't shown over the content.